releases/latest/kernel/helpers/build/populate_scc.sh

```diff
@@ -95,7 +95,7 @@ done
 # restore umask
 umask ${OLD_UMASK}
 
-rm -rf /output/messaging /logs/* $WLP_OUTPUT_DIR/.classCache
+rm -rf /output/messaging /logs/* $WLP_OUTPUT_DIR/.classCache && chmod -R g+rwx /opt/ol/wlp/output/*
```

Comment:
This is effectively going to undo the goal of https://github.com/OpenLiberty/ci.docker/pull/166, which was to not touch the permissions of the pre-existing SCC layers so that they wouldn't get pulled into the current Docker image layer.

From what I understand of this problem we want `chmod -R g+rwx /opt/ol/wlp/output/*` to be applied to all newly created files, but it also hits pre-existing files.

Reply:
hey @ymanton - from John's scenario there was a generated file during warmup start/stop which was created and had the wrong permissions.

Reply:
Yes, I assume the Liberty and umask are not enough to cover all files as Liberty still generates files owned only by user (eg ltpa.keys)
In general afaik the files that already have correct permissions will not affect final size. Or at least that can be achieved by using `find` command

Reply:
I agree with @ymanton, using chmod duplicates the files which is what #166 attempted to prevent.
It is pretty easy to verify that.
I created openliberty docker image with and without this PR.

`openliberty-chmod` is the docker image with this PR.
`openliberty` is the docker image without this PR.

```
$ docker images |grep openliberty
openliberty-chmod                    latest                    1f6f6b275408        11 minutes ago      429MB
openliberty                          latest                    4b10aba14b26        22 minutes ago      429MB
```

As can be seen, size of both is same.
Now check the size of the application docker image created using these two images as base image:
```
system-openliberty-chmod-latest      latest                    96cbc493b5d7        10 minutes ago      477MB
system-openliberty-latest            latest                    c063c90064e4        17 minutes ago      450MB
```
`system-openliberty-chmod-latest` is the image based on `openliberty-chmod`
and `system-openliberty-latest` is based on `openliberty`
You can see the size has increased by 27 MB with the change in this PR.

Another way is to check the `pmap` output of the JVM processes by creating two different apps based on same OL image.
I have another app docker image `inventory-openliberty-chmod-latest` based on `openliberty-chmod`.
I started both the docker images `system-openliberty-chmod-latest` and `inventory-openliberty-chmod-latest` and got the pmap output using `sudo pmap -X <pid>`:

```
pmap of JVM process started by system app:

         Address Perm   Offset Device     Inode    Size    Rss    Pss Referenced Anonymous LazyFree ShmemPmdMapped Shared_Hugetlb Private_Hugetlb Swap SwapPss Locked Mapping
    7f0edd300000 r--s 00000000  fd:00 143857016   26624  16640  16640      16640         0        0              0              0               0    0       0      0 C290M4F1A64P_liberty_G41L00

pmap of JVM process started by inventory app:

         Address Perm   Offset Device     Inode    Size    Rss    Pss Referenced Anonymous LazyFree ShmemPmdMapped Shared_Hugetlb Private_Hugetlb Swap SwapPss Locked Mapping
    7f8751200000 r--s 00000000  fd:00   1739287   26624  16128  16128      16128         0        0              0              0               0    0       0      0 C290M4F1A64P_liberty_G41L00
```

As can be seen the RSS and PSS column for both have same value.
If the file is being shared, the PSS would have been half of RSS.

btw I am doing these tests on an ubuntu 18 VM with following docker verison:
```
$ docker version
Client: Docker Engine - Community
 Version:           19.03.4
 API version:       1.40
 Go version:        go1.12.10
 Git commit:        9013bf583a
 Built:             Fri Oct 18 15:54:09 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.4
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.10
  Git commit:       9013bf583a
  Built:            Fri Oct 18 15:52:40 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

In essence, this PR will undo the effect of #166 . 
Seeing all these problems in managing SCC properly to get the desired memory savings, I feel it would be lot easier if we change the location of SCC to a directory which won't be touched by OL at all.